src/apps/barnbridge-smart-alpha/common/barnbridge-smart-alpha.junior-pool-token-fetcher.ts

```diff
@@ -0,0 +1,69 @@
+import { Inject } from '@nestjs/common';
```

Comment:
Rename file to `barnbridge-smart-alpha.junior-pool.token-fetcher.ts`
Same for the senior pools.